site/content/tutorial/01-introduction/05-nested-components/text.md

```diff
@@ -19,4 +19,10 @@ Add a `<script>` tag that imports `Nested.svelte`...
 <Nested/>
 ```
 
-Notice that even though `Nested.svelte` has a `<p>` element, the styles from `App.svelte` don't leak in.
\ No newline at end of file
+Notice that even though `Nested.svelte` has a `<p>` element, the styles from `App.svelte` don't leak in.
+
+**NOTE**: The component variable names (such as `Nested`) should be Capitalized
```

Comment:
```suggestion
Notice that the component name `Nested` is capitalized. This format has been adopted to provide a differentiation between user-defined components and regular HTML tags.
```

Reply:
Thank you for your review. I have added your suggestions.